api/envoy/service/load_stats/v2/lrs.proto

```diff
@@ -66,8 +66,14 @@ message LoadStatsRequest {
 // [#not-implemented-hide:] Not configuration. TBD how to doc proto APIs.
 message LoadStatsResponse {
   // Clusters to report stats for.
+  // Not populated if *send_all_clusters* is true.
   repeated string clusters = 1 [(validate.rules).repeated = {min_items: 1}];
```

Comment:
I think you need to remove the validation?

Reply:
Done.